js/components/closed-caption-multi-audio-menu/closedCaptionMultiAudioMenu.js

```diff
@@ -0,0 +1,115 @@
+var React = require('react');
+var classnames = require('classnames');
+var CONSTANTS = require('../../constants/constants');
+var Tab = require('./tab');
+var MultiAudioTab = require('./multiAudioTab');
+
+var ClosedCaptionMultiAudioMenu = React.createClass({
```

Comment:
How do we handle the old Closed Captions menu now that we have this screen? Is the old menu disabled when multi audio is available?

Reply:
I think both menus are available at this moment and they both work interchangeably, it's just that the old one provides far more customisation options (color, opacity, font size etc.), while the new one provides only language selection mechanism.